### PR TITLE
Improved Raft over XMPP ProtoXEP

### PR DESCRIPTION
**Pull request by pmembrey**
Hi,

This PR addresses a couple of issues highlighted by the Council:

- Ideally XEP shouldn't use Camel Case
- Added a section on the justification for using <message/> rather than <iq/> stanzas

Thanks!

Cheers,

Pete

### CODE REVIEW

File: inbox/raft.xml [hunk 1]
```diff
@@ -52,6 +52,16 @@
 
   <p>These are all things that would traditionally have to be re-engineered each time somebody wanted to use Raft across the public Internet. By supporting Raft in XMPP, developers looking to use Raft would have a transport layer that's as easy to use and understand as the Raft protocol itself. As Raft does not offer its own transport protocol and has deliberately left that to the developer, there is no conflict in standardizing an XMPP based transport layer.</p>
 
+  <section2 topic='Why &MESSAGE; and not &IQ;'>
+  <p>The Raft algorithm can be categorized as a request/response protocol. Normally this would make it a prime candidate for using &IQ; stanzas to handle the communication. However because Raft is designed to cope with message loss, it intrinsically supports automatic recovery. There is no need for the transport layer to report errors as even if the transport layer provided them (such as an &IQ; 'error' response), the Raft implementation cannot use it.</p>
```
Comment (Flowdalic) [inline inbox/raft.xml]: request-response

**Comment by pmembrey** [comment]
Updated ProtoXEP with feedback.

**Comment by linuxwolf** [comment]
The Editor will get the update published shortly.  Note that we will collapse the branch down, add a `<revision/>` entry for that update, and announce with an email that will look a lot like the first one.

**Comment by linuxwolf** [comment]
Merged as 87e19dd4418b42b0c901610ecfb804cc78937425.

**Pull request closed without merging**